plugins/org.eclipse.papyrus.uml.diagram.sequence.runtime/src/org/eclipse/papyrus/uml/diagram/sequence/runtime/internal/edit/policies/InteractionLayoutEditPolicy.java

```diff
@@ -102,7 +103,9 @@ protected void showLayoutTargetFeedback(Request request) {
 			Dimension proposedSize = createRequest.getSize();
 			if (proposedSize == null) {
 				// This will be null until the user draws out a rect
-				proposedSize = new Dimension(45, 180); // TODO
+				int minWidth = getLayoutConstraints().getMinimumHeight(__getHostView(this));
```

Comment:
If the `__getHostView` method isn't meant to be private to the interface type then it should be named simply `getHostView`.